backend/src/controller/board.ts

```diff
@@ -0,0 +1,20 @@
+import express from 'express';
+import { Types } from 'mongoose';
+import { boardService } from '../service/board';
+
+const router = express.Router();
+
+// Update task status
+
+router.patch('/tasks/:id', async (req, res) => {
+  try {
+    const ID = Types.ObjectId(req.params.id);
+    const body = req.body;
+    const task = await boardService.updateStatus(ID, body);
+    return res.status(201).send(task);
+  } catch (e) {
+    return res.status(400).send(e);
```

Comment:
It's not a good idea to return the whole stack-trace to the client. It may cause security issues.
Better to send only some message